rskj-core/src/main/java/co/rsk/pcc/bto/GetMultisigScriptHash.java

```diff
@@ -70,8 +75,8 @@ public Object execute(Object[] arguments) {
             throw new NativeContractIllegalArgumentException("Minimum required signatures must be greater than zero");
         }
 
-        if (publicKeys.length == 0) {
-            throw new NativeContractIllegalArgumentException("At least one public key is required");
+        if (publicKeys.length < MINIMUM_REQUIRED_KEYS) {
+            throw new NativeContractIllegalArgumentException("At least two public keys are required");
```

Comment:
just being nitpicky here. Could we just render the value of MINIMUM_REQUIRED_KEYS instead of hardcoding `two`. If by chance someone would change the constant in the future and doesn't modify this message, would be confusing.

Reply:
Good catch, updated.